lib/routes/account.js

```diff
@@ -276,7 +277,8 @@ module.exports = function (
                   uid: account.uid,
                   kA: account.kA,
                   wrapKb: wrapKb,
-                  emailVerified: account.emailVerified
+                  emailVerified: account.emailVerified,
+                  tokenVerificationId: crypto.randomBytes(32)
```

Comment:
That's correct. Both `sessionToken` and `keyFetchToken` should have the same `tokenVerificationId`.